baseimages/scanner/cli/root.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cli
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	debug bool
+)
+
+const globalUsageMessage = `Welcome to Scanner - Azure Container Registry's dependency scanner.
+
+To start working with Scanner, run scanner --help
+`
+
+// Execute executes the root command.
+func Execute() {
+	cmd := newRootCmd(os.Args[1:])
+	if err := cmd.Execute(); err != nil {
+		os.Exit(1)
+	}
+}
+
+func newRootCmd(args []string) *cobra.Command {
+
+	cmd := &cobra.Command{
```

Comment:
just curious as to why  go lang "flag" didn't work here?

Reply:
It does, this just gives you a lot of extra functionality.